### DISCUSSION TITLE
Thanks, a great little tool

### DISCUSSION THREAD

**Discussion by valeriupredoi** [flowwer-dev/pull-request-stats#39] [General] [answered]
Hey fellas, just wanted to drop a line and say thanks, a very cool tool! Cheers :beer:

**Comment by manuelmhtr** [comment, marked as answer]
Hi @valeriupredoi thank you very much! 🍻 
It's a side project I started because needed to increase the reviews speed on my team. I'm glad you like it and willing to keep improving!
